graphql_schema.json

```diff
@@ -2187,6 +2407,22 @@
                   "ofType": null
                 }
               },
+              "isDeprecated": true,
+              "deprecationReason": "use creatorAccount field instead"
+            },
+            {
+              "name": "creatorAccount",
```

Comment:
possible to use `producerAccount` or something? `creatorAccount` is really close to `createAccount`

Reply:
I can't imagine people would get confused there. It's more a question of whether "creator" is an inappropriate name?